pkg/providers/amifamily/bootstrap/windows.go

```diff
@@ -17,16 +17,39 @@ package bootstrap
 import (
 	"bytes"
 	"encoding/base64"
+	"encoding/xml"
 	"fmt"
+	"html"
 	"strings"
+
+	"github.com/samber/lo"
 )
 
 type Windows struct {
 	Options
 }
 
+type UserDataXML struct {
```

Comment:
Just so I understand, these are all of the possible fields that can be specified in the Windows UserData XML?

Reply:
With the exception of one more arg `runAsLocalSystem`, the list is exhaustive yes. I will add the remaining field `runAsLocalSystem`.

Reply:
Removed the flexibility to specify EC2Launch XML elements. In the future if the need arises we can implement further changes to support this.